src/main/resources/db/migration/V46__oppgave.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE OPPGAVE (
+    id bigint primary key,
+    fk_behandling_id bigint references behandling (id) not null,
+    gsak_id varchar(50) not null,
+    type varchar(100) not null,
```

Comment:
```suggestion
    gsak_id varchar not null,
    type varchar not null,
```

Varchar er 512 by default, men bruker bare lengden av strengen. Foreslår derfor å fjerne eksplisitt lengde.